src/v/archival/ntp_archiver_service.cc

```diff
@@ -996,14 +1029,15 @@ ss::future<bool> ntp_archiver::maybe_upload_manifest(const char* upload_ctx) {
         co_return false;
     }
 
-    // Do not upload if interval has not elapsed.  The upload loop will call
+    // Do not upload if interval has not elapsed. The upload loop will call
     // us again periodically and we will eventually upload when we pass the
-    // interval.  Make an exception if we are under local storage pressure,
-    // and skip the interval wait in this case.
-    if (
-      !local_storage_pressure() && _manifest_upload_interval().has_value()
-      && ss::lowres_clock::now() - _last_manifest_upload_time
-           < _manifest_upload_interval().value()) {
+    // interval. Make an exception if we are under local storage pressure and
+    // skip the interval wait in these cases.
+    bool below_manifest_upload_interval
+      = _manifest_upload_interval().has_value()
+        && ss::lowres_clock::now() - _last_manifest_upload_time
+             < _manifest_upload_interval().value();
+    if (!local_storage_pressure() && below_manifest_upload_interval) {
```

Comment:
did something change here? it looks like a refactor, but it's hard to tell.

if it is a refactor, you can make it easier for reviewers by doing one of these:

1. split out into a separate commit with a message that says "refactor"
2. review your own PR and add an inline comment like this one here that says "this is a refactor"

Reply:
Removed edits here, they were an artifact of old changes, but yes, they were entirely refactors and non-functional.

Well noted re: notes on refactor and helping reviewers in the future :+1: